Libraries/Components/TextInput/TextInput.js

```diff
@@ -537,7 +537,9 @@ var TextInput = React.createClass({
   },
 
   _onPress: function(event: Event) {
-    this.focus();
+    if (!this.props.editable || this.props.editable === undefined) {
```

Comment:
Maybe I have the logic backwards in my head somehow but it seems like the `!` before `this.props.editable` should not be there - `editable` as false would return true for it, and then focus, `editable` as true would return false and not focus.

```
if (this.props.editable || !this.props.hasOwnProperty('editable')) {
  this.focus();
}
```